packages/cf-component-box/README.md

```diff
@@ -0,0 +1,53 @@
+# cf-component-box
+
+> Cloudflare Box Component
+
+This is styling component similar to cf-component-flex which allows for arbitrary styling.
+
+This component should only be used when the built in style doesn't quite fit the bill and when
+it can't be done with fela. Ideally, cf-ui components should have the right styling built in for
+most cases. In the few edge cases that pop up in real world feature development, this is an alternative
+to wrapping your component in a `<div>` or `<span>` and styling it with css to accomodate one-offs.
+
+It supports the following styling properties:
+  * margin
+  * padding
+  * border
+    * borderWidth
+    * borderColor
+    * borderStyle
+  * backgroundColor
```

Comment:
The common use case for the Box could be to display an image. In that case we could consider adding:
- backgroundImage
- backgroundRepeat
- backgroundPositionX
- backgroundImagePositionY

It's definitely could be added as a separate PR, just share my thoughts.